src/documents/firstDiscoveryServer.md

```diff
@@ -0,0 +1,383 @@
+
+(See: [Starting a Kettle application](https://github.com/amb26/kettle/blob/KETTLE-32/README.md#starting-a-kettle-application))
+
+### Programmatically ###
+
+```javascript
+// require the kettle module
+var kettle = require("kettle");
+
+// load the config
+kettle.config.loadConfig({
+    // path to the config directory
+    configPath:"./src/config",
+
+    // name of the config to load, without the file extension
+    configName: "vagrant"
+});
+```
+
+### Command Line ####
+
+```bash
+# Call Kettle's init.js script with the
+# configPath and configName
+# node node_modules/kettle/init.js <configPath> [<configName>]
+node node_modules/kettle/init.js ./src/config vagrant
+
+# or using an environment variable to specify
+# the configName
+# NODE_EVN=<configName> node node_modules/kettle/init.js <configPath>
+NODE_EVN=vagrant node node_modules/kettle/init.js ./src/config
+```
+
+## REST API ##
+
+<table>
+    <thead>
+        <tr>
+            <th>URL</th>
+            <th>Request</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>/user[?view=:view]</code></td>
+            <td><code>POST</code></td>
+            <td>
+                Accepts a set of preferences, in a JSON object, to be stored on the preferences server. For example:
+
+                <pre>
+<code>{"gpii_firstDiscovery_language": "en-US"}</code>
+                </pre>
+
+                On successfully storing the preferences set, a JSON object is returned containing `userToken` and `preferences` properties. The `userToken` is the GPII token, which can be used for retrieving the preferences on a GPII enabled device. The `preferences` are a confirmation of the preferences stored on Preferences server for that GPII token. For example:
+
+                <pre>
+
+<code>{
+  "userToken": "2288e676-d0bb-4d29-8131-7cff268ba012",
+  "preferences": {
+    "contexts": {
+      "gpii-default": {
+        "name": "Default preferences",
+        "preferences": {
+          "gpii_firstDiscovery_language": "en-US"
+        }
+      }
+    }
+  }
+}</code>
+                </pre>
+
+                The optional <code>view</code> query parameter is used to specify which ontology the preferences are stored in. (See: <a href="https://github.com/GPII/universal/blob/master/documentation/PreferencesServer.md#post-preferencesviewview">Preferences Server</a>)
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## Demos
+
+This demo implements the integration with the [GPII Preferences Server](https://github.com/GPII/universal/blob/master/documentation/PreferencesServer.md) to save preferences.
+http://build.fluidproject.org/first-discovery/demos/prefsServerIntegration
+
+## Source code
+
+* [Client (First Discovery Tool)](https://github.com/GPII/first-discovery)
+* [Server (First Discovery Server)](https://github.com/GPII/first-discovery-server)
+* [GPII Preferences Server](https://github.com/GPII/universal)
```

Comment:
This link can be a little more specific, since the user might take a while to locate the prefs server within the great bulk of the universal project